### ISSUE TITLE
Can't visualize prediction images 

### ISSUE THREAD

**Issue by Didier0** [lih0905/PyTorch_Study#1]
When I try this: `python setup.py build_ext install `
error shows me: ```
```
fatal error: pycocotools/_mask.c: No such file or directory
compilation terminated.
error: command '/home/landeka/anaconda3/envs/Model1/bin/gcc' failed with exit code 1
```

I don't know how did you visualize prediction images, I tried everything with loading models with `torch.load` on 3 ways and still nothing. 

And tried to do this:
```
img, _ = dataset_test[20]
# put the model in evaluation mode
model.eval()
#evaluate(model, data_loader_test, device=device)
with torch.no_grad():
    prediction = model([img.to(device)])
    #print(prediction)
Image.fromarray(img.mul(255).permute(1, 2, 0).byte().numpy())
Image.fromarray(prediction[0]['masks'][0, 0].mul(255).byte().cpu().numpy())
```

But what is your `model `? This -> model = get_model_instance_segmentation(num_classes) ??
